client/deposits/instant-deposits/index.js

```diff
@@ -14,10 +14,32 @@ import './style.scss';
 import InstantDepositModal from './modal';
 import { useInstantDeposit } from 'data';
 
-const InstantDepositButton = ( {
-	balance: { amount, fee, net, transaction_ids: transactionIds },
-} ) => {
+const getModalVars = ( overview ) => {
+	if ( overview && overview.instant_balance ) {
+		const instant = overview.instant_balance;
```

Comment:
I can tweak the server so, if the merchant is eligible for Instant Deposits, but they can't do it right now (limit reached for the day, insufficient funds, etc), the server will return an "empty" `instant_balance`, like this:
```
instant_balance: {
    amount: 0,
    fee: 0,
    net: 0,
    transaction_ids: []
}
```

Does that sound good? You can then check if `amount === 0` to disable the button.

Reply:
Yup, that works 👍

Reply:
There you go, update the `feature/instant-deposits` branch: https://github.com/Automattic/woocommerce-payments-server/pull/455/commits/fb6ab8893c2f6884a974559f82d28c46dac6d2de